### ISSUE TITLE
Bot stops working

### ISSUE THREAD

**Issue by Tayfinford** [helionmusic/rhinobot_heroku#44]
**Describe the problem** 
Bot stops working after a couple of hours of use.
**Have you followed all the instructions?** 
- [X] Yes, I followed the instructions properly.

**Tried updating the dependencies?** 
- [X] Yes, they are up to date.
- [ ] Not applicable for this issue.

**Bot version** 
MusicBot release-260819 Heroku Edition

Logs:

2020-06-19T01:03:37.100261+00:00 app[worker.1]: [31m[ERROR:bot] Exception in on_message
2020-06-19T01:03:37.100263+00:00 app[worker.1]: Traceback (most recent call last):
2020-06-19T01:03:37.100263+00:00 app[worker.1]:   File "/app/musicbot/bot.py", line 2821, in on_message
2020-06-19T01:03:37.100263+00:00 app[worker.1]:     response = await handler(**handler_kwargs)
2020-06-19T01:03:37.100264+00:00 app[worker.1]:   File "/app/musicbot/bot.py", line 1328, in cmd_play
2020-06-19T01:03:37.100264+00:00 app[worker.1]:     await self.send_typing(channel)
2020-06-19T01:03:37.100264+00:00 app[worker.1]:   File "/app/musicbot/bot.py", line 881, in send_typing
2020-06-19T01:03:37.100264+00:00 app[worker.1]:     return await destination.trigger_typing()
2020-06-19T01:03:37.100265+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.7/site-packages/discord/abc.py", line 839, in trigger_typing
2020-06-19T01:03:37.100265+00:00 app[worker.1]:     await self._state.http.send_typing(channel.id)
2020-06-19T01:03:37.100269+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.7/site-packages/discord/http.py", line 158, in request
2020-06-19T01:03:37.100270+00:00 app[worker.1]:     async with self.__session.request(method, url, **kwargs) as r:
2020-06-19T01:03:37.100270+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.7/site-packages/aiohttp/client.py", line 1005, in __aenter__
2020-06-19T01:03:37.100271+00:00 app[worker.1]:     self._resp = await self._coro
2020-06-19T01:03:37.100271+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.7/site-packages/aiohttp/client.py", line 497, in _request
2020-06-19T01:03:37.100273+00:00 app[worker.1]:     await resp.start(conn)
2020-06-19T01:03:37.100273+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.7/site-packages/aiohttp/client_reqrep.py", line 844, in start
2020-06-19T01:03:37.100273+00:00 app[worker.1]:     message, payload = await self._protocol.read()  # type: ignore  # noqa
2020-06-19T01:03:37.100274+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.7/site-packages/aiohttp/streams.py", line 588, in read
2020-06-19T01:03:37.100274+00:00 app[worker.1]:     await self._waiter
2020-06-19T01:03:37.100274+00:00 app[worker.1]: aiohttp.client_exceptions.ClientOSError: [Errno 104] Connection reset by peer[0m

**Comment by LegendModzYT** [comment]
Have you enabled the option on Heroku to make the bot auto-restart if this happens so the bot will just restart itself and fix it. It works for me most of the time but haven't had any problems since editing it myself so yea hoping this fixes your problem.

**Comment by codeinteger6** [comment]
I have updated the bot to release-010620u, try it and see if it solves your issue.

Starting from this release, we have switched to the unofficial fork of the bot maintained by Team-JSB, which has a few developers from the original team and has been promoted in the official Discord support server. As this release contains several fixes as well as some new features, we recommend you update to this release. The fork will be updated regularly unlike the main repo.

To prevent any issues while updating, we strongly suggest you delete and recreate the Heroku application from scratch.

@LegendModzYT, AFAIK there is no option to enable/disable auto restart. As per Heroku policy all applications are restarted every 24 hours.
